src/components/tab-group/tab-group.scss

```diff
@@ -0,0 +1,61 @@
+@import 'variables';
+@import 'default-theme';
+
+:host {
+  display: block;
+  font-family: sans-serif;
```

Comment:
Seems like maybe this should use the common `$font-family` variable? Or should this font be different?